src/components/Button/components/RegularButton/__snapshots__/RegularButton.spec.js.snap

```diff
@@ -727,6 +736,7 @@ exports[`RegularButton should have secondary flat button styles 1`] = `
   font-weight: 700;
   width: auto;
   height: auto;
+  text-align: center;
```

Comment:
I think this should already be here on master. Or was there some issue with filtering the props before? Was this accidentally getting filtered? I remember updating the snapshot. 😕